.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- Write a description of what the PR solves and how -->
+
+<!-- Link to relevant Jira issue, add multiple if necessary -->
+Jira Issue: [RHELC-]()

```

Comment:
Do we still need to place the Jira issue in the description? I thought that we dropped it in favor of having it in the title because of the integration...

Reply:
While true, and it seems like it is the case where we need it in the title (unless the tool just didn't work when I tested) it still helps to have a link somewhere to quickly get to an issue

Reply:
If we need to have the [RHELC-xy] in the PR title then it can be another item in the checklist:
`[ ] [RHELC-] is part of the PR title <!-- For a proper sync with Jira -->`